app/code/Magento/CatalogSearch/Model/Search/FilterMapper/TermDropdownStrategy/StockConditionApplier.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\CatalogSearch\Model\Search\FilterMapper\TermDropdownStrategy;
+
+use Magento\Framework\App\ResourceConnection;
+use Magento\Framework\DB\Select;
+
+/**
+ * Apply stock condition to select.
+ */
+class StockConditionApplier
```

Comment:
rename the class to `ApplyStockConditionToSelect`

Reply:
Done.